frontend/tests/controllers/ContestUsersTest.php

```diff
@@ -551,6 +551,92 @@ public function testContestPracticeWhenOriginalContestHasNotEnded() {
         }
     }
 
+    public function testNextRegisteredContestForUser() {
+        // Create 2 users
+        ['user' => $user1, 'identity' => $identity1] = \OmegaUp\Test\Factories\User::createUser(new \OmegaUp\Test\Factories\UserParams([
+            'username' => 'test_contest_user_1',
+        ]));
+        ['user' => $user2, 'identity' => $identity2] = \OmegaUp\Test\Factories\User::createUser(new \OmegaUp\Test\Factories\UserParams([
+            'username' => 'test_contest_user_2',
+        ]));
+
+        // Get the next registered contest for a user who is not registered in any contest
+        $nextRegisteredContestForUser1 = \OmegaUp\Controllers\Contest::getNextRegisteredContestForUser(
+            $identity1
+        );
+        $this->assertEmpty($nextRegisteredContestForUser1);
+
+        $timePast =  new \OmegaUp\Timestamp(\OmegaUp\Time::get() - 120 * 60);
+        $timeFuture1 =  new \OmegaUp\Timestamp(\OmegaUp\Time::get() + 60 * 60);
+        $timeFuture2 =  new \OmegaUp\Timestamp(\OmegaUp\Time::get() + 120 * 60);
```

Comment:
Acá como no estás cambiando la hora, ¿qué te parece si sólo la obtienes una vez y la guardas en una variable?

```suggestion
        $currentTime = \OmegaUp\Time::get();

        $timePast =  new \OmegaUp\Timestamp($currentTime - 120 * 60);
        $timeFuture1 = new \OmegaUp\Timestamp($currentTime + 60 * 60);
        $timeFuture2 = new \OmegaUp\Timestamp($currentTime + 120 * 60);
```